src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -64,6 +72,8 @@ internal TextEditor(TextEditor handle, bool shown = true) : this(Interop.TextEdi
             {
                 SetVisible(false);
             }
+
+            initializeTextEditor();
```

Comment:
This is duplicate of line:86

Reply:
The whole initializeText****() functions are not necessary if we decide to do not use null-able property.
I will remove it.